internals/models.py

```diff
@@ -554,6 +554,30 @@ def _annotate_first_of_milestones(self, feature_list, version=None):
     except Exception as e:
       logging.error(e)
 
+  @classmethod
+  def _annotate_first_of_impl_status_in_milestones(self, feature_list, version=None):
+    try:
+      versions = [
```

Comment:
"versions" sounds like it would be milestones.  "statuses" would be a better name.

Reply:
True. Thanks for pointing this out. Done.